nsis/install.nsi

```diff
@@ -423,6 +425,11 @@ SectionGroup $(TITLE_SEC_DOCS) SEC06
     SetOutPath "$INSTDIR\share\doc\kicad\help\pl"
     File /nonfatal /r "..\share\doc\kicad\help\pl\*"
   SectionEnd
+    Section $(LANGUAGE_NAME_ZH) SEC06_ZH
```

Comment:
Please fix the indentation, that is remove the two spaces

Reply:
OK,I've fixed it.